policies/attribute_name_collisions.rego

```diff
@@ -72,13 +78,17 @@ to_const_name(name) = const_name if {
     const_name := replace(name, ".", "_")
 }
 
-# These lists contain exceptions for existing collisions that were introduced unintentionally.
+get_or_null(obj, key) = value if {
```

Comment:
nit: Should this be: "has_or_null"?

I.e. we check existence and turn it into a true or false of sorts.